rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -69,14 +65,29 @@ class RecordFixture : public TemporaryDirectoryFixture
     rclcpp::shutdown();
   }
 
+  std::string get_bag_file_name(int split_index) const
+  {
+    std::stringstream bag_file_name;
+    bag_file_name << "bag_" << split_index;
+
+    return bag_file_name.str();
+  }
+
+  rcpputils::fs::path get_bag_file_path(int split_index)
+  {
+    return root_bag_path_ / (get_bag_file_name(split_index) + ".db3");
```

Comment:
This assumes an SQLite storage plugin only.

Reply:
Correct, however the tests only work on sqlite storage plugin.

Reply:
I see that it's just moving from https://github.com/ros2/rosbag2/pull/286/files#diff-3609b823f2d4cae7c0418a8efe81cc96L50 - it's out of scope for this PR but we should think about how this test can run against arbitrary plugins

Reply:
@zmichaels11 can you file an issue?